# Automatically Generated in 2017-06-12 22:33:47.453178
# MESSAGE: CONFIG_PARAM_FLOAT
# Description:Config parameter information for float param
uint8 ID = 199
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
char[30] name
float32 value
float32 default_value
float32 min
float32 max
